src/main/java/org/mockito/junit/MockitoJUnitRunner.java

```diff
@@ -152,7 +152,7 @@ public MockitoJUnitRunner(Class<?> klass) throws InvocationTargetException {
         this(new StrictRunner(new RunnerFactory().createStrict(klass), klass));
     }
 
-    MockitoJUnitRunner(InternalRunner runner) throws InvocationTargetException {
+    protected MockitoJUnitRunner(InternalRunner runner) throws InvocationTargetException {
```

Comment:
Curious why changing the public modifier?

I suspect that this change may not be binary compatible in case somebody extends the runner.